examples/try-locally.md

```diff
@@ -88,7 +88,7 @@ spec:
       annotations:
```

Comment:
```suggestion
      annotations:
          secrets-webhook.security.bank-vaults.io/provider: "vault"
```

Shouldn't this annotation be here as well?

Reply:
yes, it should be thanks for catching this! ^^